sources/netdog/src/networkd/config/netdev.rs

```diff
@@ -142,7 +142,7 @@ impl Display for NetDevMacAddress {
 }
 
 impl NetDevConfig {
-    const FILE_EXT: &str = "netdev";
+    const FILE_EXT: &'static str = "netdev";
```

Comment:
Unnecessary, literals/consts are always understood to be static.

```suggestion
    const FILE_EXT: &str = "netdev";
```